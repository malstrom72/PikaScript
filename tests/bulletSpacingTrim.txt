- Apple
  tastes good

  indeed

1) Banana
   is tasty

   yes indeed

*) Carrot
   is crunchy

   yes
