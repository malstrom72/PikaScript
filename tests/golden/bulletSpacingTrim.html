
<ul>
<li>Apple<br>tastes good
<br><br>indeed
</li>
</ul>

<ol>
<li>Banana<br>is tasty
<br><br>yes indeed
</li>
</ol>

<ul>
<li>Carrot<br>is crunchy
<br><br>yes
</li>
</ul>

